src/ucm/api/ucm.h

```diff
@@ -8,17 +8,17 @@
 #ifndef UCM_H_
 #define UCM_H_
 
-#include <ucs/sys/compiler.h>
+#include <ucs/sys/compiler_def.h>
 
 BEGIN_C_DECLS
```

Comment:
@yosefe Perhaps it is better to remove this line and don't provide `BEGIN_C_DECLS` macro to the user level.